tests/Api/OrdersTest.php

```diff
@@ -30,10 +30,10 @@ public function test_can_call_magento_api_orders_all_with_filter()
             '*rest/all/V1/orders*' => Http::response([], 200),
         ]);
 
-        $api = MagentoFacade::api('orders')->all(1, 1, [
+        $api = MagentoFacade::api('orders')->all([
```

Comment:
I see that this is also to resolve issues created by changing the order of arguments in the ID fork.

Reply:
@developh 
Please discard the changes to this test because the order of arguments for `Orders::all()` was reverted in `main` to remain compatible with the original package and therefore this test will fail if merged into `main`.